### PR TITLE
feat: added button component

### PR DESCRIPTION
**Pull request by mansi2011**
feat: added button component
- primary button
- link button
- icon button 
- floating button 

button

### CODE REVIEW

**Review by prathmeshjagtap — APPROVED** [review]

File: Components/Button/Button.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    height: 20rem;
+    width: 40rem;
+    position: relative;
+    box-shadow: 0.5rem 1rem 0.5 var(--inactive-color);
+    
+}
+
+.float-btn{
+    position: absolute;
+    bottom: 0rem;
```
Comment (SanjayJatti) [inline Components/Button/Button.css:67]: You can give bottom and right space for floating action button.

**Review by dev-enforced — APPROVED** [review]

File: Components/Button/Button.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    border: 1px solid var(--inactive-color);
+    background-color: var(--white-font-color);
+    cursor: pointer;
+}
+
+.signup-btn:hover{
+    border: 2px solid var(--pink-border-color);
+}
+
+.link-btn{
```
Comment (SanjayJatti) [inline Components/Button/Button.css:35]: Underline don't look good for links. You can use text-decoration: none.

**Review by SanjayJatti — APPROVED** [review]

File: Components/Button/Button.css (hunk 2 above)
Comment (mansi2011) [reply Components/Button/Button.css:35]: i made that explicitly.

File: Components/Button/Button.css (hunk 1 above)
Comment (mansi2011) [reply Components/Button/Button.css:67]: okay will consider that.

**Pull request merged**